src/views/TradingCompetition/components/PrizesInfo/PrizesCard.tsx

```diff
@@ -24,17 +24,15 @@ const PrizesCard = () => {
           {TranslateString(999, 'Higher trading volume = higher rank!')}
         </Text>
       </CardHeader>
-      <CardBody>
-        <PrizesGrid />
-      </CardBody>
-      <CardFooter>
+      <PrizesGrid />
+      <Box p="24px">
```

Comment:
Special case - this card doesn't use a traditional card footer.

Reply:
Ah nice. I've never used the `Box` before... just an unflexed `Flex` with access to all the non-flex props?